Add topic about how to update product attribute in bulk (#1352)
magento/merchdocs: src/stores/bulk-product-attribute-update.md

---

> title: Bulk Updating Attributes

Reviewer: ```suggestion
title: Bulk Updates for Product Attributes
```

Author: Done ✅

---

> Use the "Update Attributes" tool to change one or more attributes in your products. This tool helps to massively transform the desired parts of the products.

Reviewer: ```suggestion
Use the _Update Attributes_ tool to change one or more attributes in your products. This tool allows you to apply significant changes across a large group of products.
```

Author: Done ✅

## Steps to update product attribute in bulk

1. On the _Admin_ sidebar, go to **Catalog** > **Products**.

1. Select the products for which you want to modify sources.

   Browse or search to find the products and select those checkboxes.

1. Click the **Actions** menu at the top and choose **Update Attributes**.

    ![]({% link images/images/bulk-product-updating-action.png %}){: .zoom}

>     _Select products to attribute update_

Reviewer: ```suggestion
    _Select products to be updated_
```

Author: Done ✅

> 1. Update the attribute, advanced inventory or website data for product you want.

Reviewer: ```suggestion
1. Update the attribute, advanced inventory, or website data for the selected products, according to your needs.
```

Author: Done ✅

> 1. When finish, click <span class="btn">Save</span>.

Reviewer: ```suggestion
1. When finished, click <span class="btn">Save</span>.
```

Author: Done ✅

    ![]({% link images/images/bulk-pproduct-attribute-update.png %}){: .zoom}

>     _Bulk Updating Attribute_

Reviewer: ```suggestion
    _Bulk updating for attributes_
```

Author: Done ✅
